src/utils/Util.ts

```diff
@@ -307,3 +307,24 @@ export function checkIfNameChanged() {
         return false;
     }
 }
+
+/**
+ * reads data from local file into a JSON object
+ *
+ * @param file - the absolute file path
+ * @param defaultType - a JSON default type to return
+ */
+export function getFileDataAsJson(file: string, defaultResult: any = {}) {
```

Comment:
I was thinking more like the following. Less conditions and if anything happens with the read or parse its all caught by the catch, and we still return the default (or null) at the end. 

export function getFileDataAsJson(file: string, defaultResult: any = null) {
    if (fs.existsSync(file)) {
	    try {
	        return JSON.parse(fs.readFileSync(file, "utf-8"));
	    } catch (err) {
	        console.log("Could not read file:" + err.message);
	    }
	}
    return defaultResult;
}